Userland/Utilities/acpi.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <stdio.h>
+#include <unistd.h>
+
+// https://uefi.org/sites/default/files/resources/ACPI_Spec_6_4_Jan22.pdf#page=193
+struct [[gnu::packed]] SDTHeader {
```

Comment:
This looks like it should have a very specific size, maybe add an assertion for that, eg:
```c++
static_assert(AssertSize<SDTHeader, wanted_size_in_bytes>());
```

Reply:
The assert has been added with a reference to the spec for the magic number

Reply:
Please move it directly under the class definition,
and you don't need to declare an extra variable for the size

Reply:
Moved the assert  up
That is so much clearer. I'll use that pattern in other places